CODEOWNERS

```diff
@@ -1,6 +1,7 @@
 # CODEOWNERS: https://help.github.com/articles/about-codeowners/
 
 /chain/ @SkidanovAlex @bowenwang1996
+/chain/client/src/view_client.rs @frol @khorolets
```

Comment:
```suggestion
/chain/client/src/view_client.rs @frol @khorolets @bowenwang1996
```

Reply:
You already owner of entire `/chain/` does it makes sense to add you as owner of specific file in it? Is our change overrides your ownership?